packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -56,16 +76,37 @@ class MemoryBodyState extends State<MemoryBody> {
         Expanded(
           child: Split(
             axis: Axis.vertical,
-            firstChild: MemoryChart(memoryController),
-            secondChild: const Text('Memory Panel TBD'),
+            firstChild: MemoryChart(),
+            secondChild: const Text('Memory Panel TBD capacity'),
             initialFirstFraction: 0.25,
           ),
         ),
       ],
     );
   }
 
+  void _updateListeningState() async {
+    await serviceManager.serviceAvailable.future;
+
+    final MemoryController memoryController = Controllers.of(context).memory;
+    if (memoryController.hasStarted) return;
+
+    await memoryController.startTimeline();
+/*
```

Comment:
Remove commented code.

Reply:
Added TODO I need to make sure I do this.
      // TODO(terry): Need to set the initial state of buttons.